asn1crypto/x509.py

```diff
@@ -1528,7 +1528,7 @@ def url(self):
                 if general_name.name == 'uniform_resource_identifier':
                     url = general_name.native
                     lurl = url.lower()
-                    if lurl.startswith('http://') or lurl.startswith('https://') or lurl.startswith('ldap://'):
+                    if lurl.startswith(('http://', 'https://', 'ldap://', 'ldaps://')):
```

Comment:
Since we now only use `lurl` once, we can probably just do `url.lower().startswith()`